doc/manual/core_language_reference.rst

```diff
@@ -1,7 +1,7 @@
 Core language reference
 =======================
 
-The most commonly used features from the ARTIQ language modules and from the core device modules are bundled together in :mod:`artiq.experiment` and can be imported with ``from artiq.experiment import *``.
+The most commonly used features from the ARTIQ language modules and from the core device modules are bundled together in ``artiq.experiment`` and can be imported with ``from artiq.experiment import *``.
```

Comment:
There is no link; `artiq.experiment` isn't documented in the manual. This goes along with removing ('py:mod', 'artiq.experiment') from the `nitpicky_ignore` list. It's changed only because I made the (arbitrary) decision it's aesthetically cleaner to just have it in double backticks instead of as a dead link with an individual ignore entry, but there's no real difference either way, so it can be changed back if we prefer.